app/src/app/usePyrogTheme.ts

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+
+import { createMuiTheme, useMediaQuery, Theme } from "@material-ui/core";
+
+declare module "@material-ui/core/styles/createPalette" {
+  interface Palette {
+    badges: {
+      required: string;
+      pending: string;
+    };
+    icons: {
+      table: string;
+      disabled: string;
+      fhir: string;
```

Comment:
Does typing this with `React.CSSProperties['color']` work ? This would be better